packages/frontend/src/App.tsx

```diff
@@ -44,6 +45,18 @@ const Root = (props: RouteSectionProps<unknown>) => {
     );
 };
 
+export function TheoryHelpPage() {
```

Comment:
Can you move this function to `help_page.tsx`? I'd like to keep the main entry point in `App.tsx` as uncluttered as we can.